### PR TITLE
Added Eclipsepvp

### PR DESCRIPTION
**Pull request by JuanSeeee**
### General:
* [x] The pull request title is descriptive. *(ex. `Added Lunar Network`, not `Updated servers.json`)*
* [x] The pull request does not contain any unrelated commits.  *(ex. commits from previous pull requests)*

### Mapping Additions or Updates:
* [x] No changes were made to the file's formatting.
* [x] There are no syntax errors.
* [x] There is no pre-existing mapping matching my id.
* [x] My field values match your requirements.
* You can view our patterns here: [servers.schema.json](https://github.com/LunarClient/ServerMappings/blob/master/servers.schema.json), or take a look below and complete the field checklist:
  - [x] `id`: a lowercase string *(ex. `myserver`)*
  - [x] `name`: a string *(ex. `MyServerPvP`)*
  - [x] `addresses`: an array with lowercase strings *(ex. of address `my.server`)*
    - You do not need to specify sub-domains, Lunar Client services automatically detect them.

### Logo Additions or Updates:
* [x] My image is a `png` file.
* [x] I uploaded my image to the `logos` directory.
* [x] My image has a transparent background and is square (1:1 aspect ratio).
* [x] My image is `512` pixels in width and height.
* [x] My image's file name matches its mapping id.

### CODE REVIEW

File: servers.json [hunk 1]
```diff
@@ -1,4 +1,11 @@
 [
+    {
+        "id" : "Eclipsepvp" ,
```
Comment (jacobk999) [inline servers.json:3]: The `id` field should be lowercase. You are also missing your server logo in the `logos` directory

**Review by jacobk999 — CHANGES_REQUESTED** [review]

**Review by GrowlyX — CHANGES_REQUESTED** [review]
I don't think the other reviewers noticed this, but your field keys must be in English, matching the existing mappings.

Please change:
 - `nombre` -> `name`
 - `direcciones` -> `addresses`

File: servers.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 [
     {
         "id" : "eclipsepvp",
```
Comment (GrowlyX) [inline servers.json:3]: Could you also remove the unnecessary whitespaces after the field names?

**Review by GrowlyX — CHANGES_REQUESTED** [review]

**Review by GrowlyX — DISMISSED** [review]

**Review by GrowlyX — APPROVED** [review]

**Comment by GrowlyX** [comment]
Your server logo is still missing.

**Comment by jacobk999** [comment]
Closed for inactivity.

**Pull request closed without merging**